Sorting/selection_sort/selection_sort.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+
+using namespace std;
+
+void swapElement(int *m, int *n)  
+{  
+    int temp = *m;  
+    *m = *n;  
+    *n = temp;  
+}  
+  
+void selectionSort(int arr[], int n)  
+{  
+    int i, j, minVal;  
+  
+    // Loop unsorted Array values 
+    for (i = 0; i < n-1; i++)  
+    {  
+        // Find the lowest value  
+        minVal = i;  
+        for (j = i+1; j < n; j++)  
+        if (arr[j] < arr[minVal])  
+            minVal = j;  
```

Comment:
Since `minVal` is storing the index of minimum value and not the minimum value itself, can we change the comments and variable name to clarify?

Reply:
Thanks for Reviewing @nikochiko . I'll make the suggested changes.